src/helm/benchmark/presentation/run_specs_chat_gpt.conf

```diff
@@ -134,26 +136,27 @@ entries: [
   # ChatGPT sometimes responds that it can't do complex reasoning problems.
```

Comment:
it might be interesting to note if this "it cant do complex reasoning" depends mostly on the subject rather than the level. Bit minor, but I imagine some folks might find interesting